### PR TITLE
Update fabfile to work with EuroPython server and containers.

### PR DESCRIPTION
**Pull request by dadoeyad**
We need to put a username that everyone knows their password and maybe change the container name if it's required.

### CODE REVIEW

**Comment by alexsavio** [comment]
Oh gosh, we left this unwatched. Do we still need this?

**Comment by patrick91** [comment]
@alexsavio I think so :) we have some informations on how to update a container here: https://eps.projecthut.com/trac/europython/wiki/Web%20WG/Updating%20a%20website%20checkout%20in%20a%20running%20container

a fabric file would be totally helpful :) also most of us have users on the server, so we are fine, we just need to pass the user and pass via a parameter (or use ssh keys)

**Comment by malemburg** [comment]
+1 on having such common tasks in the fabfile.py - even if only to document them.

**Comment by LuRsT** [comment]
This is mega old, close?

File: fabfile.py [hunk 1]
```diff
@@ -1,213 +1,86 @@
 from fabric.api import env
 
-env.user = 'pyconwww'
-env.hosts = ['pycon.it']
+env.user = 'eyad'
```
Comment (alexsavio) [inline fabfile.py:3]: I we still want to merge this, we should change env.user, right?

**Comment by alexsavio** [comment]
Any plans to merge this or should we close it?

**Comment by malemburg** [comment]
We're not using fabric anymore, so I think we can just remove it.

**Pull request closed without merging**